docs/layers/identity/deploy.mdx

```diff
@@ -62,28 +62,36 @@ These components are expected to be used together to provide fine-grained role d
     ```
 
     This will generate AWS configuration to the `rootfs` directory locally.
+
+    <Note title="IMPORTANT">
+    The script that generates AWS config requires local files that are created when the AWS Team and Team Roles components are deployed.
+
+    If you did not run these steps yourself and they are not committed to git, then you will need to rerun those components to generate these local files. For this reason, we recommend committing these generated files to git so that they are available for future use.
+    </Note>
   </Step>
 
   <Step>
     ### <StepNumber/> Apply Changes to Geodesic
 
     In order to apply these updates to your Geodesic
     image, exit the image and rerun `make all`.
-  </Step>
-</Steps>
 
-One of these configuration files will be named `aws-config-teams`. This file holds local configuration for all local
-user authentication to AWS. This file will be the same for both SAML and SSO users, as well as for all teams. This file
-looks for a specific AWS profile for your namespace, for example `acme-identity `, and uses that profile as the
-entry-point for all role assumption. From this profile, all users with proper permission will be able to execute
-Terraform or assume other roles across the AWS Organization.
+    One of these configuration files will be named `aws-config-teams`. This file holds local configuration for all local
+    user authentication to AWS. This file will be the same for both SAML and SSO users, as well as for all teams. This file
+    looks for a specific AWS profile for your namespace, for example `acme-identity `, and uses that profile as the
+    entry-point for all role assumption. From this profile, all users with proper permission will be able to execute
+    Terraform or assume other roles across the AWS Organization.
 
-:::info
+    :::info
```

Comment:
See my note about admonitions.

Reply:
Odd, seems like it's working for you though. Anyways, if you have trouble, convert it to `<Admonition/>` tags.